Update installation-user-infra-machines-static-network.adoc (#58631)
openshift/openshift-docs: modules/installation-user-infra-machines-static-network.adoc

vlan=bond0.100:bond0
----

> 

Reviewer: It would be useful to add an`id` metadata entry here. E.g. something like:
```suggestion
[id="bonding-multiple-network-interfaces-with-mtu_{context}"]
```

Author: Done.

[discrete]

> === Bonding multiple network interfaces to a single interface with mtu 

Reviewer: I think MTU should be capitalized here:
```suggestion
=== Bonding multiple network interfaces to a single interface with an MTU 
```

Author: Done.

> 

Reviewer: There needs to be some descriptive text here, to keep it consistent with the other examples in this section. What do you think about the following:
```suggestion
Optional: You can configure an MTU on a VLAN subinterface by adding `bond0:none:9000` to the `ip=` parameter to specify the MTU, for example:
```

Author: Done.

Reviewer: @Rupesh-git-eng I don't see the description update in the commit history, did you push the changes?

Author: Sorry, looks like it got lost during the update. Can you check it once again, please?

https://github.com/Rupesh-git-eng/openshift-docs/edit/patch-8/modules/installation-user-infra-machines-static-network.adoc?pr=%2Fopenshift%2Fopenshift-docs%2Fpull%2F58631

[id="bonding-multiple-network-interfaces-with-mtu_{context}"]
[discrete]

> === Optional: You can configure the maximum transmission unit (MTU) on a BOND interface by adding `bond0:none:9000` to the `ip=` parameter to specify the MTU, for example: 

Reviewer: ```suggestion
=== Bonding multiple network interfaces to a single interface with an MTU
Optional: You can configure the maximum transmission unit (MTU) on a bonded interface by adding `bond0:none:9000` to the `ip=` parameter to specify the MTU, for example: 
```
